runtime/rococo/src/xcm_config.rs

```diff
@@ -120,6 +121,9 @@ match_types! {
 	pub type OnlyParachains: impl Contains<MultiLocation> = {
 		MultiLocation { parents: 0, interior: X1(Parachain(_)) }
 	};
+	pub type SystemParachains: impl Contains<MultiLocation> = {
+		MultiLocation { parents: 0, interior: X1(Parachain(1000 | 1002)) }
```

Comment:
We could move these numbers to constants like on the other runtimes to avoid having magic numbers